_sources/configs/robots.php

```diff
@@ -28,3 +28,10 @@
 }
 
 readfile( 'robots.txt' );
+
+// If the file `extra-robots.txt` is created under the name
+// `/var/www/mediawiki/extra-robots.txt` then its contents get appended to the
+// default `robots.txt` - see MW-312
+if ( file_exists( 'extra-robots.txt' ) && is_readable( 'extra-robots.txt' ) ) {
+	readfile( 'extra-robots.txt' );
```

Comment:
```suggestion
        echo "\n";
	readfile( 'extra-robots.txt' );
```
But is there like a way to just prepend each to robots.txt

Reply:
We don't need to check that robots.txt exists since it is part of Taqasta and thus will always be there